### ISSUE TITLE
live stream fall out

### ISSUE THREAD

**Issue by tzarebczan** [lbryio/lbry-desktop#5777]
### Next Phase

- [x] remote url processing A - needs api from live stream service to list replays.  (@DispatchCommit )
- [x] add to level 5 channels, use new "odysee_live_enabled" instead of experiemntal ui, and block when odysee_live_disabled present. (@seanyesmunt )
- [x] remote url processing B - Allow picker if more than 1 replay exists. Pass this as POST param (like you would a file) but don't pass the file, instead pass `remote_url`. Preview should still work without the remote url being passed (preview what?)
- [x] fix chat scroll issue (stops auto scrolling) (@DispatchCommit)
- [x] when user goes live, call live stream notification internal api to trigger notification (@DispatchCommit)
- [ ] remove bitwave references to code + apis (needs niko?) (@DispatchCommit )
- [x] include everyone level 5 in livestreaming (@seanyesmunt)
- [x] call view endpoint when stream is actually live (or on page load if that's easier and it's not live) (@seanyesmunt )
- [x] ensure nothing shows on bitwave (@DispatchCommit )
- [x] Fix disappearing pending claim in confirmation mode - claim search is called right after claim list and it disappears (@jessopb)
- [x] Fix disappearing pending claim on live stream edit
- [x] fix disappearing pending claim on live stream create (this doesn't happen all the time, but still does - claim search runs after and does fix it, but you are in a weird state until then) 
- [x] Bad publish state (@jessopb)
- [x] firefox black video bug
- [x] reposts show up under pending on live stream dashboard (they also don't have a source, unless we reference the originalclaim)
- [x] add instructions to dashboard (working on this with Juilian)
- [x] if you are on a channel page that had a pending claim or hasn't created a claim, it won't recheck if the channel is live. 
- [x] only call streaming service to check if live if there are claims with no-stream on the channel
- [x] improve edit flow - should say Publish Stream instead of edit on the file page. 
- [x] smooth out stream create > dashboard flow

### After Next Phase
- [ ] if a claim is pending on one channel, other channels will not see an option to make a live stream claim.
- [ ] if live stream + replay/file published, update release time to current timestamp (the original may have been created well before the stream)
- [x] when going to channel page, it should check for live streams + api right away (there's a delay now - especially bad if coming from a notification which you'd expect to see it)
- [ ] keep polling on channel page for live streams (30s, 1m?). Right now you won't see it go live if you are already on the page.
- [x] can't click live stream tile outside of the thumbnail on the channel page when live (i.e. clicking it doesn't take you to the stream, have to click thumb)
- [x] #6492
- [x] on claim list view mode, you can see the live streams while not live (hidden in tile mode)
- [x] when selecting go live on mobile, encourage users to get apps (and give sadface to Android users for now)
- [x] hide stream key by default (someone recently pulled it up during stream) 
- [x] for users without a channel, if they try to chat, pop them out to channel creation in a new tab (today it leaves the stream)
- [x] Make sure edit flow goes to upload page when we add remote url (today it goes to dashboard when you don't pass a file). Also make sure there are not duplicate pending claims after this.  
- [x] on mobile, sometimes you need to click play on the stock video to play the livestream. 
- [ ] view with no channels: 
![image](https://user-images.githubusercontent.com/8120721/114081784-fe9bfd80-987a-11eb-89be-e8469e43c888.png)
- [x] fix OG metadata for livestreams (they should be playable when linked on Twitter)

**Comment by DispatchCommit** [comment]
The firefox bug was an issue when using some settings with privacy mode that blocked 3rd party cookies.

I've resolved the issues caused by 3rd party cookies being blocked, but cannot support cases in which *all* cookies are blocked.

In cases where all cookies are blocked by the client, I've added some error / help text instructing viewers to enable cookies. This is done by adding the text via pre-rendering on the server, and removing the element once the client has `mounted`.

Here's what it looks like: 
![image](https://user-images.githubusercontent.com/45262335/113109428-d34e4a00-91ba-11eb-9797-4dd688bad01e.png)
